lib/plausible_web/components/billing.ex

```diff
@@ -317,6 +305,48 @@ defmodule PlausibleWeb.Components.Billing do
 
   def subscription_paused_notice(assigns), do: ~H""
 
+  def private_preview_end_notice(assigns) do
+    user = assigns.user |> Plausible.Users.with_subscription()
+
+    features_to_lose =
+      case Plans.get_subscription_plan(user.subscription) do
+        nil ->
+          []
+
+        %Plan{kind: :business} ->
+          []
+
+        _free_10k_or_enterprise_or_growth ->
+          used_features = Plausible.Billing.Quota.features_usage(assigns.user)
+          for f_mod <- [Funnels, RevenueGoals], f_mod in used_features, do: f_mod
```

Comment:
More of a matter of preference perhaps, but I personally got a little pause trying to sort out what's going on here 😅 Maybe this would communicate intent a bit clearer?

```suggestion
          Enum.filter([Funnels, RevenueGoals], &(&1 in used_features))
```

Reply:
Oh yeah I totally overdid this... Looks much better with `Enum.filter`, thanks! :) https://github.com/plausible/analytics/pull/3510/commits/20f4a2df97c5201ed114c8f7e61a595aec915276